docs/CashFlowStatementsApi.md

```diff
@@ -50,7 +51,7 @@ namespace Example
 
             try
             {
-                PaginatedCashFlowStatementList result = apiInstance.CashFlowStatementsList(xAccountToken, createdAfter, createdBefore, cursor, includeRemoteData, modifiedAfter, modifiedBefore, pageSize, remoteId);
+                PaginatedCashFlowStatementList result = apiInstance.CashFlowStatementsList(xAccountToken, createdAfter, createdBefore, cursor, includeDeletedData, includeRemoteData, modifiedAfter, modifiedBefore, pageSize, remoteId);
```

Comment:
do we know if these param changes are breaking?

Reply:
They are, in order to do this in a backwards compatible way we'd need to wrap these parameters in a request-options class or builder. I can make it version 1.1.0 instead, but I'm saving 2.0 for the unified category sdk